apps/marketplace/pages/SellerAssessmentViewPage.js

```diff
@@ -0,0 +1,144 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { actions } from 'react-redux-form'
+import { Redirect } from 'react-router-dom'
+import formProps from 'shared/form/formPropsSelector'
+import SellerAssessmentStages from 'marketplace/components/SellerAssessment/SellerAssessmentStages'
+import { rootPath } from 'marketplace/routes'
+import { loadDomainData, loadEvidenceData, saveEvidence } from 'marketplace/actions/supplierActions'
+import SellerAssessmentView from 'marketplace/components/SellerAssessment/SellerAssessmentView'
+import LoadingIndicatorFullPage from 'shared/LoadingIndicatorFullPage/LoadingIndicatorFullPage'
+import { SellerAssessmentFormReducer, SellerAssessmentEvidenceReducer } from 'marketplace/reducers'
+
+const model = 'SellerAssessmentForm'
```

Comment:
Given that `SellerAssessmentView` is not actually a form but a read-only view of the evidence submission, I wonder if this page could be much simpler, and not reference the form model at all. `getEvidenceData()` would not need to prepare the form model and call `changeFormModel` (so it wouldn't have to do anything, just calling `getEvidenceData()` would be enough to populate `this.props.evidence`), and the `meta={{ domain: this.props.domain, evidence: this.props.evidence }}` property passed to `SellerAssessmentView` would actually be all the data that view needs.

Reply:
Resolved